common/startup/check_database_connection.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function check_database_connection {
+  local IS_MYSQL_ALIVE=false
+  local FAILURE_COUNT=0
+  local MAX_FAILURE_COUNT=31
+
+  while [[ "${IS_MYSQL_ALIVE}" != "mysqld is alive" ]]; do
+    FAILURE_COUNT=$((FAILURE_COUNT + 1))
+    if [[ $FAILURE_COUNT -gt $MAX_FAILURE_COUNT ]]; then
+    echo "----- Startup Checks Error Found -----" >&2
+      echo "MySQL is not responding after ${MAX_FAILURE_COUNT} attempts. Exiting." >&2
+      echo "Please ensure the MySQL server is running and accessible from this container." >&2
+      exit 1
+    fi
+
+    if [[ "${IS_MYSQL_ALIVE}" =~ "error" ]]; then
+      echo "Error: MySQL response contained error: ${IS_MYSQL_ALIVE}" >&2
```

Comment:
```suggestion
      echo "----- Startup Checks Error Found -----" >&2
      echo "MySQL response contained error: ${IS_MYSQL_ALIVE}" >&2
```

Improve consistency between error logs

Reply:
Understood the approach  now